src/Backend.hs

```diff
@@ -44,20 +44,32 @@ type DelegationsApi = GetJ DelegationNetwork
 delegationsApi :: (GenArbitrary m, ActionM m) => ServerT DelegationsApi m
 delegationsApi = Action.loginByName "admin" >> fishDelegationNetworkAction Nothing
 
+data NeedEmptyUserMap = NeedEmptyUserMap
+
+instance Page NeedEmptyUserMap where
+    isAuthorized = publicPage
+{-
+    FIXME: since we don't have access to the db here.
+    isAuthorized _ = do
+        noUsers <- query $ views dbUserMap Map.null
+        if noUsers
+            then accessGranted
+            else accessDenied $ Just "Can only be used when there are no users!"
+-}
```

Comment:
In thentos, we have a credential *is coming from localhost*.  I like this better because it doesn't require any additional work to avoid having to race attackers after deployment.  For now I'd like to leave this a FIXME, but I'll add this thought to it.